modules/create-cluster-logging-cli.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-deploying.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-cluster-logging-cli_{context}"]
+= Creating a ClusterLogging object by using the CLI
+
+This default {logging} configuration supports a wide array of environments. Review the topics on tuning and configuring components for information about modifications you can make.
+
+.Prerequisites
+
+* You have installed the {clo}.
```

Comment:
You have installed the {clo} and the {eo} since we are creating ES too on the below ClusterLogging instance.

Reply:
Added it for now. The intention eventually would be to have both Loki and ES documented here, or have a generic CL CR.